packages/shared/src/UI/components/SocialAccountList/SocialListItem.tsx

```diff
@@ -0,0 +1,219 @@
+        marginTop: 2,
+        display: 'flex',
+        alignItems: 'center',
+        whiteSpace: 'nowrap',
+        overflow: 'auto',
+    },
+    addressText: {
+        fontSize: '10px',
+        overflow: 'auto',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        '::-webkit-scrollbar': {
+            display: 'none',
+        },
+    },
+    copyButton: {
+        marginLeft: theme.spacing(0.5),
+        color: MaskColors.light.text.secondary,
+        padding: 0,
+    },
+    followButton: {
+        marginLeft: 'auto',
+        height: 32,
+        padding: theme.spacing(1, 1.5),
+        backgroundColor: '#ABFE2C',
+        color: '#000',
+        borderRadius: 99,
+        minWidth: 64,
+        fontSize: 12,
+        '&:hover': {
+            backgroundColor: '#ABFE2C',
+            color: '#000',
+        },
+    },
+    linkIcon: {
+        display: 'flex',
+        marginLeft: 'auto',
+    },
+    linkOutIcon: {
+        cursor: 'pointer',
+    },
+    related: {
+        whiteSpace: 'break-spaces',
+        lineBreak: 'anywhere',
+        display: 'flex',
+        flexWrap: 'wrap',
+        gap: 4,
+    },
+    ens: {
+        whiteSpace: 'nowrap',
+        padding: theme.spacing(0.25, 0.5),
+        marginRight: 6,
+        fontSize: 12,
+        maxWidth: '100%',
+        display: 'inline-block',
+        textOverflow: 'ellipsis',
+        overflow: 'hidden',
+        backgroundColor: theme.palette.maskColor.bottom,
+        borderRadius: 4,
+    },
+}))
+
+const ENSAddress = memo(({ domain }: { domain: string }) => {
+    const { classes } = useStyles()
+    const { value: address } = useAsync(() => {
+        return ENS.lookup(domain)
+    }, [domain])
+
+    if (!address) return null
+
+    return (
+        <div className={classes.address}>
+            <Typography className={classes.addressText}>{address}</Typography>
+            <CopyButton text={address} size={14} className={classes.copyButton} />
+        </div>
+    )
+})
+ENSAddress.displayName = 'ENSAddress'
+
+interface SocialAccountListItemProps {
+    platform: NextIDPlatform
+    identity: string
+    name: string
+    onClose: () => void
+    relatedList?: BindingProof[]
+    link?: string
+}
+
+export function SocialAccountListItem({
+    platform,
+    link,
+    identity,
+    name,
+    onClose,
+    relatedList,
+}: SocialAccountListItemProps) {
+    const t = useSharedI18N()
+    const { account } = useChainContext()
+    const { classes, cx } = useStyles()
+    const Icon = resolveNextIDPlatformIcon(platform)
+
+    const { loading, value } = useAsync(async () => {
+        if (platform !== NextIDPlatform.LENS || !identity) return
+        const profile = await Lens.getProfileByHandle(identity)
+        const isFollowing = await Lens.queryFollowStatus(account, profile.id)
+        return {
+            ownedBy: profile.ownedBy,
+            isFollowing,
+        }
+    }, [identity, platform, account])
```

Comment:
Critical dependency.
```suggestion
    }, [identity, platform !== NextIDPlatform.LENS, account])
```

Reply:
There are other platform judgments in this component, such as ens, so I used the first example